packages/squiggle-lang/src/rescript/Distributions/GenericDist/GenericDist.res

```diff
@@ -154,140 +154,141 @@ let truncate = Truncate.run
   TODO: It would be useful to be able to pass in a paramater to get this to run either with convolution or monte carlo.
```

Comment:
I think my last PR on this should have deleted this TODO.

Reply:
Also see `2` in docstring: should be `sqrt(2)`

Reply:
> Also see 2 in docstring: should be sqrt(2)
I don't follow? What does this refer to?